Gemfile

```diff
@@ -135,6 +135,9 @@ group :development, :test do
   gem "rubocop-rspec", require: false
   gem "rbs_rails"
   gem "steep"
+  # Adds support for Solargraph LSP
+  gem 'solargraph-rails', '~> 0.3.1'
+  gem "undercover"
```

Comment:
Unrelated ?

Reply:
yes, while resolving conflicts, this has come by mistake
will remove that